src/components/Sidebar.vue

```diff
@@ -170,6 +195,22 @@ export default {
 				})
 		},
 
+		onUpdateTitle(newTitle) {
+			this.newTitle = newTitle
+		},
+
+		onRenameTitle(event) {
+			if (event && this.title !== this.newTitle) {
```

Comment:
Is  it possible that an event is triggered, that is not defined ?
I think it is safe to remove `event` from the if clause